src/script_opt/Inline.cc

```diff
@@ -166,7 +166,8 @@ void Inliner::Analyze() {
         inline_ables[func] = f.Profile();
     }
 
-    CoalesceEventHandlers();
+    if ( ! analysis_options.no_eh_coalescence )
+        CoalesceEventHandlers();
```

Comment:
Could've squashed that in the commit that added the option, too.

Reply:
> Could've squashed that in the commit that added the option, too.

Oops yeah not sure how that wound up being separate.